tests/ext/trace_method_that_will_exist_later.phpt

```diff
@@ -0,0 +1,25 @@
+--TEST--
+Check that a function can be traced before it exists.
```

Comment:
This file was renamed to **tests/ext/trace_method_or_function_that_will_exist_later.phpt** in #325 so it can be deleted. :)